pkg/syncer/k8scloudoperator/k8scloudoperator.go

```diff
@@ -286,11 +287,12 @@ func (k8sCloudOperator *k8sCloudOperator) PlacePersistenceVolumeClaim(ctx contex
 		return out, err
 	}
 
-	scName := pvc.Spec.StorageClassName
-	if scName == nil || *scName == "" {
-		return out, nil
+	scName, err := syncer.GetSCNameFromPVC(pvc)
+	if err != nil {
+		return out, err
```

Comment:
Can you log the error message using `log.Errorf` before returning err, just like we have been doing at all other places.

Reply:
done :)